PINCE.py

```diff
@@ -1790,9 +1791,8 @@ def __init__(self, parent, description=tr.NO_DESCRIPTION, address="0x", value_ty
         guiutils.fill_value_combobox(self.comboBox_ValueType, vt.value_index)
         guiutils.fill_endianness_combobox(self.comboBox_Endianness, vt.endian)
         self.lineEdit_Description.setText(description)
-        self.offsetsList = []
-        self.offsetDerefLabels = []
-        self.offsetTextLabels = []
+        self.lineEdit_Description.setFixedWidth(180)
+        self.offsetsList:list[PointerChainOffset] = []
```

Comment:
We should have the type hints syntax more consistent across the codebase so I suggest using `self.offsetsList: list[PointerChainOffset]` instead of having no space after `:` which makes it easier to be missed.

Reply:
This applies to everywhere you used type hints in this PR, just to be explicit

Reply:
(we need auto formatting for stuff like that.)